src/iperf_api.c

```diff
@@ -705,7 +709,12 @@ iperf_parse_arguments(struct iperf_test *test, int argc, char **argv)
     char *client_username = NULL, *client_rsa_public_key = NULL;
 #endif /* HAVE_SSL */
 
-    while ((flag = getopt_long(argc, argv, "p:f:i:D1VJvsc:ub:t:n:k:l:P:Rw:B:M:N46S:L:ZO:F:A:T:C:dI:hX:", longopts, NULL)) != -1) {
+    char optstring[] = "p:f:i:D1VJvsc:ub:t:n:k:l:P:Rw:B:M:N46S:L:ZO:F:A:T:C:dI:hX:";
+#ifdef HAVE_GETOPT_H
+    while ((flag = getopt_long(argc, argv, optstring, longopts, NULL)) != -1) {
+#else
+    while ((flag = netbsd_getopt_long(argc, argv, optstring, longopts, NULL)) != -1) {
```

Comment:
The other thing is, why use 2 different prefixes of `netbsd_` versus `missing_`?  Seems confusing.